egs/librispeech/s5/local/chain/run_chain_common.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+
+# this script has common stages shared across librispeech chain recipes
+set -e
+
+# configs for 'chain'
+stage=0
+# chain options
+frames_per_eg=150
+max_wer=
+
+# output directory names
+dir=
+treedir=
+lang=
+min_seg_len=
+# End configuration section.
+echo "$0 $@"  # Print the command line for logging
+
+. cmd.sh
```

Comment:
you should source cmd.sh in the current directory. The cmd.sh file might not be available in the path.

so `. cmd.sh` should be changed to `. ./cmd.sh`.

This change was pointed out yesterday, but you might have missed the comment. You could actually monitor the conversation in github to find the latest changes.